### PR TITLE
bacula-fd 7.4.2

### PR DESCRIPTION
**Pull request by ilovezfs**
Also,
- don't set user name in the plist
- add a test

### CODE REVIEW

File: Formula/bacula-fd.rb [hunk 1]
```diff
@@ -18,24 +18,19 @@ def install
...
 
     system "make"
     system "make", "install"
 
-    # Ensure var/run exists:
-    (var + "run").mkpath
-
-    # Create the working directory:
-    (var + "lib/bacula").mkpath
+    (var/"run").mkpath
```
Comment (DomT4) [inline Formula/bacula-fd.rb]: These should be shunted into `post_install`.

Comment (ilovezfs) [inline Formula/bacula-fd.rb]: Can you explain the logic for those given they're in the .bottle hidden dir?

Comment (ilovezfs) [inline Formula/bacula-fd.rb]: Or is it because it's var/run not var/run/bacula so it doesn't make it in there?

File: Formula/bacula-fd.rb [hunk 2]
```diff
@@ -18,24 +18,19 @@ def install
     #   (conio support not tested)
     # * working directory in /var/lib/bacula, reasonable place that
     #   matches Debian's location.
-    readline = Formula["readline"].opt_prefix
     system "./configure", "--prefix=#{prefix}",
-                          "--sbindir=#{bin}",
                           "--with-working-dir=#{var}/lib/bacula",
-                          "--with-pid-dir=#{HOMEBREW_PREFIX}/var/run",
+                          "--with-pid-dir=#{var}",
```
Comment (ilovezfs) [inline Formula/bacula-fd.rb]: I suppose I shouldn't have deleted the "run" there.

File: Formula/bacula-fd.rb (hunk 1 above)
Comment (DomT4) [inline Formula/bacula-fd.rb]: At this point, no. I remembered it once, but I didn't retain the reasoning. You'll have to ask @MikeMcQuaid.

Comment (ilovezfs) [inline Formula/bacula-fd.rb]: OK, thanks.

**Comment by ilovezfs** [comment]
PR refreshed.

File: Formula/bacula-fd.rb [hunk 3]
```diff
@@ -18,24 +18,22 @@ def install
     #   (conio support not tested)
     # * working directory in /var/lib/bacula, reasonable place that
     #   matches Debian's location.
-    readline = Formula["readline"].opt_prefix
     system "./configure", "--prefix=#{prefix}",
-                          "--sbindir=#{bin}",
```
Comment (MikeMcQuaid) [inline Formula/bacula-fd.rb]: I'd rather we left this as-is for now rather than janking around current users.

**Comment by MikeMcQuaid** [comment]
One comment otherwise 👍

File: Formula/bacula-fd.rb (hunk 3 above)
Comment (ilovezfs) [inline Formula/bacula-fd.rb]: Not that discussion again!
*ilovezfs curls up into the fetal position*

Comment (MikeMcQuaid) [inline Formula/bacula-fd.rb]: Sorry! I mean until we've decided on a resolution to that discussion: let's leave formulae that are using `bin`/`sbin` as-is.

Comment (ilovezfs) [inline Formula/bacula-fd.rb]: OK.

**Comment by ilovezfs** [comment]
PR refreshed.

**Comment by DomT4** [comment]
:shipit:

**Comment by MikeMcQuaid** [comment]
👍

**Pull request closed without merging**